SampleProject/src/main/java/pl/edu/agh/mwo/App.java

```diff
@@ -0,0 +1,7 @@
+package pl.edu.agh.mwo;
```

Comment:
Package name 'pl.edu.agh.mwo' does not correspond to the file path 'SampleProject.src.main.java.pl.edu.agh.mwo'